paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -433,6 +433,10 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
     private suspend fun retrieveSavedPaymentMethodSelection(
         config: PaymentSheet.Configuration,
     ): SavedSelection.PaymentMethod? {
+        /*
+         * We only care is the `SavedSelection` is a `PaymentMethod` type and not `Google Pay` or `Link`. Therefore
```

Comment:
`SavedSelection` is a `PaymentMethod` is a little confusing. Maybe reword this a bit?

Also, is it worth calling out anything about elements/sessions here? Elements sessions is the one reading this, right?